Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -194,6 +196,26 @@ internal class CustomerIOImplementation: CustomerIOInstance {
         name: String,
         data: RequestBody
     ) {
+        // check for profile identified first before other checks. This is to prevent the scenario where
+        // we save the last tracked screen name, then identify, then we might ignore a screen view event
+        // because of saving the last tracked screen name already.
+        guard profileStore.identifier != nil else {
+            // XXX: when we have anonymous profiles in SDK,
+            // we can decide to not ignore events when a profile is not logged yet.
+            logger.info("ignoring screen \(name) because no profile currently identified")
+            return
+        }
+
+        // this logic is to prevent duplication. Sometimes with method swizzling, you can receive multiple calls to the
+        // SDK for the same 1 screen. Only track an event if it's unique.
+        if let previousScreenTrackedName = globalDataStore.lastTrackedScreenName, previousScreenTrackedName == name {
```

Comment:
think this could be simplified to `if globalDataStore.lastTrackedScreenName == name {` instead